### ISSUE TITLE
Context Awareness breaks dates

### ISSUE THREAD

**Issue by nitricware** [rgamba/tonic#4]
Hy!

It's not possible to insert a european style date.

``` php
{$time.toLocal().date("d. m. Y")}
```

breaks the code as a whole and

``` php
{$time.toLocal().date("d m Y")}
```

returns 01+02+2015

So I tried a workaround. Creating a date like 01. 02. 2015 also creates 01.+02.+2015

Disabling context awareness fixed the issue. But I'd like it enabled.

Is there any setting I'm missing?

UPDATE

Funny - today, without changing the code, it works with context awareness enabled...
- Kurt

**Comment by rgamba** [comment]
Great, looks like it's caused by the same thing as issue #5 , it breaks with the colon.
Thanks!

**Comment by nitricware** [comment]
You're welcome! Glad I could help!

**Comment by rgamba** [comment]
Fixed in commit [b1dc7a32](https://github.com/rgamba/tonic/commit/b1dc7a32c6ab74a8c05743c2ff3a55b225fe770e)
Colons are now allowed inside modifier parameters
